src/views/EditEnemy/EnemyEdit.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router';
+import AdminForm from '../../components/Admin/Admin';
+import { fetchEnemyId, updateEnemy } from '../../services/Enemies';
+
+export default function EnemyEdit(props) {
+  const [enemy, setEnemy] = useState({});
+
+  const history = useHistory();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchEnemyId(props.match.params.id);
```

Comment:
It might make sense to set id to it's own variable at the top of the function as we are using it in multiple places